etc/profile-a-l/cloneit.profile

```diff
@@ -0,0 +1,63 @@
+# Firejail profile for cloneit
+# Description: A CLI tool to download specific GitHub directories or files
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include cloneit.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist /usr/libexec
+blacklist ${RUNUSER}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+whitelist ${DOWNLOADS}
+include whitelist-common.inc
```

Comment:
```suggestion
```

This could be used for development (that is, downloading files to a
user-specific development directory), so I'd avoid whitelisting anything in
`${HOME}`.

Reply:
Done!